Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -0,0 +1,19 @@
+module RuboCop
+  module Cop
+    class FormulaCop < Cop
+      @registry = Cop.registry
+
+      def file_inside_formula_root?
+        return false unless processed_source.path =~ %r{/Taps/homebrew}
+        true
```

Comment:
Actually, this may not be necessary. We can use `Include` and `Exclude` options for individual cops in `.rubocops.yml` to pass file/directory patterns to decide whether cops will run or not. I will update the PR. 
@reitermarkus Where else are the Formulae located?